app/models/miq_queue.rb

```diff
@@ -364,6 +364,7 @@ def deliver(requester = nil)
       _log.error("#{MiqQueue.format_short_log_msg(self)}, #{message}")
       status = STATUS_EXPIRED
     rescue Exception => error
+      raise if error.class.name == "IRB::Abort"
```

Comment:
@jrafanie possibly maybe the above (or even what @kbrock has), is what we do if we want to backport this, but what you suggested is what we do moving forward in the next release.  The above basically maintains the current functionality, and that shouldn't change if we backport this, but we really should do what you suggested moving forward.

We should probably document that this in the CHANGELOG specifically that this is coming in a new release, and that custom ruby executed code might break/cause weird issues if they were raising errors not subclassed by `StandardError`.